### ISSUE TITLE
allow pinning multiple tags at once

### ISSUE THREAD

**Issue by hoijui** [tvo/rapid#1]
```
rapid pin ct:stable ct:test
```

or

```
rapid pin-all ct:stable ct:test
```

**Comment by tvo** [comment]
Makes sense, and should be easier/safer to do now I'm using optparse.

Will see if I can put that in some version, for most commands.

**Comment by hoijui** [comment]
nice! :-)

**Comment by tvo** [comment]
pin, unpin, install and uninstall now support arbitrary amount of arguments

Rapid simply loops through the arguments executing the desired operation
for each of them.  A line "---" is displayed between each two iterations.

Additionally, select and TUI.choose_many don't call sys.exit anymore when
something is wrong, but instead return an empty selection.  This allows
rapid to continue with the next argument (if any).

Closed by cae195153ef3a5e2decf0311e058e7ab9ab582cd
